test/services/cognitiveServicesVision/faceAPIClient-tests.js

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+
+const should = require('should');
+const util = require('util');
+
+const CognitiveServicesCredentials = require('ms-rest-azure').CognitiveServicesCredentials;
+
+const SuiteBase = require('../../framework/suite-base');
+const FaceAPIClient = require('../../../lib/services/cognitiveServicesVision/lib/face/faceAPIClient');
+const fs = require('fs');
+
+let requiredEnvironment = [
+  { name: 'AZURE_FACE_KEY', secure: true }
+];
+
+let testPrefix = 'face-tests';
+let suite;
+let client;
+let credentials;
+
+describe('Face', function () {
+  before(function (done) {
+    suite = new SuiteBase(this, testPrefix, requiredEnvironment);
+    suite.setupSuite(function () {
+      credentials = new CognitiveServicesCredentials(process.env["AZURE_FACE_KEY"]);
```

Comment:
Can you make sure to document all the required environment variables for Cognitive Services tests in [this](https://github.com/Azure/azure-sdk-for-node/blob/master/Documentation/EnvironmentVariables.md) file by creating a new section called `#### CognitiveServices"